inc/theme-hooks.php

```diff
@@ -125,3 +125,111 @@ function alcatraz_output_footer_bottom() {
 		);
 	}
 }
+
+
+add_filter( 'alcatraz_primary_sidebar', 'alcatraz_sidebar_nav_menu', 5);
+/**
+ * Output the sidebar nav.
+ *
+ * @since  1.0.0
+ */
+
+function alcatraz_sidebar_nav_menu() {
+
+	global $post;
+
+	$args = array();
+
+	//defaults
+	$defaults = array(
+		'icon'             => "<i class='fa fa-chevron-right sidebar-icon'></i>",
```

Comment:
Because we're creating a theme meant to be extended through a child theme, I think we'd get the most extensibility through using a `::before` pseudo element in the styles.